_data/events.yml

```diff
@@ -1,6 +1,4 @@
 dates:
-  - Tuesday March 3rd @10am
-  - Tuesday March 10th @12pm
```

Comment:
As mentioned, lets leave March 10 as the room is still booked.

Reply:
Done